docs/EDID to Receiver Caps Mapping.md

```diff
@@ -86,17 +86,31 @@ CVT 3 Byte Code structure is defined in [E-EDID][E-EDID] section 3.10.3.8.
+#### Short Video Descriptors
 
-Video Data Block is defined in [CTA-861][CTA-861] section 7.5.1.
+Short Video Descriptor (SVD) format is defined in [CTA-861][CTA-861] section 7.5.1.
 
-It operates with Video Identification Codes (VICs), each of them is associated with a union of frame width, height and rate and interlace mode. This mapping is defined in [CTA-861][CTA-861] section 4.1.
+It operates with Video Identification Codes (VICs). Each of them is associated with a union of frame width, height and rate and interlace mode. This mapping is defined in [CTA-861][CTA-861] section 4.1.
 
-Some of VICs are marked as associated with two flavours of the same mode: with a frame rate that is an integer multiple of 6 Hz and a frame rate adjusted by a factor of 1000/1001. Such VICs MUST be described with `urn:x-nmos:cap:format:grain_rate` supporting both frame rates.
+### Vertical Frequency Discrepancy
 
-### Color subsampling
+[E-EDID][E-EDID] describes video timings which actual vertical frequency may not be equal to the grain rate of the corresponding video mode.
 
-If EDID doesn't have the [CTA-861][CTA-861] Extension Block, color subsampling formats MUST be taken from Base EDID, otherwise from the Extenstion Block.
+Established Timings, Standard Timings and CVT 3 Byte Codes operate with video mode descriptions with integer vertical frequencies although these video modes correspond to video timings with fractional vertical frequencies (some of these timings are described in [DMT][DMT]).
+
+Describing such video modes in Receiver Capabilities is implementation specific and depend on what network stream the Receiver can handle. `urn:x-nmos:cap:format:grain_rate` in corresponding Constraint Set(s) MUST describe vertical frequency from either the video mode or the video timings and MAY describe both.
+
+Some VICs ([CTA-861][CTA-861] section 4.1) are marked as associated with two flavours of the same mode: with a vertical frequency that is an integer multiple of 6 Hz and a vertical frequency adjusted by a factor of 1000/1001. `urn:x-nmos:cap:format:grain_rate` in corresponding Constraint Set(s) MUST describe at least one of these vertical frequencies and MAY describe both.
+
+### Video Mode Preference
+
+Constraint Sets for Detailed Timing Descriptors and Short Video Descriptors describing _Native Video Formats_ MUST have higher `urn:x-nmos:cap:meta:preference` values than Constraint Sets for video modes not marked as native.
+
+The Constraint Set for the first Detailed Timing Descriptor in Base EDID, called _Preferred Timing Mode_, or the Constraint Set for the first Short Video Descriptor in the first CTA-861 Extension if it takes precedence ([CTA-861][CTA-861] section 7.5) MUST have the highest `urn:x-nmos:cap:meta:preference` value among the Constraint Sets.
+
+### Color Subsampling
+
+If an [E-EDID][E-EDID] doesn't have any [CTA-861][CTA-861] Extensions, color subsampling formats MUST be taken from Base EDID, otherwise from the Extenstions.
```

Comment:
```suggestion
If an [E-EDID][E-EDID] doesn't have any [CTA-861][CTA-861] Extensions, color subsampling formats MUST be taken from Base EDID, otherwise from the Extensions.
```